### ISSUE TITLE
 failed install with 'apm install script'

### ISSUE THREAD

**Issue by genghuiluo** [atom-ide-community/atom-script#1012]
verbose mode msg

```
REQUEST end event https://registry.npmjs.org/es6-symbol
REQUEST emitting complete https://registry.npmjs.org/es6-symbol
npm ERR! git clone --template=C:\Users\geluo\.atom\.apm\_git-remotes\_templates --mirror git://github.com/monsterkodi/node-simple-plist.git C:\Users\geluo\.atom\.apm\_git-remotes\git-github-com-monsterkodi-node-simple-plist-git-6a71a541: Cloning into bare repository 'C:\Users\geluo\.atom\.apm\_git-remotes\git-github-com-monsterkodi-node-simple-plist-git-6a71a541'...
npm ERR! git clone --template=C:\Users\geluo\.atom\.apm\_git-remotes\_templates --mirror git://github.com/monsterkodi/node-simple-plist.git C:\Users\geluo\.atom\.apm\_git-remotes\git-github-com-monsterkodi-node-simple-plist-git-6a71a541: fatal: read error: Software caused connection abort
npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Users\\geluo\\AppData\\Local\\atom\\app-1.8.0\\resources\\app\\apm\\bin\\node.exe" "C:\\Users\\geluo\\AppData\\Local\\atom\\app-1.8.0\\resources\\app\\apm\\node_modules\\npm\\bin\\npm-cli.js" "--globalconfig" "C:\\Users\\geluo\\.atom\\.apm\\.apmrc" "--userconfig" "C:\\Users\\geluo\\.atom\\.apmrc" "install" "D:\\cygwin_root\\tmp\\d-116621-22568-1yf57n1\\package.tgz" "--target=0.36.8" "--arch=ia32"
npm ERR! node v0.10.40
npm ERR! npm  v2.13.3
npm ERR! code 128

npm ERR! Command failed: Cloning into bare repository 'C:\Users\geluo\.atom\.apm\_git-remotes\git-github-com-monsterkodi-node-simple-plist-git-6a71a541'...
npm ERR! fatal: read error: Software caused connection abort
npm ERR!
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>
```

**Comment by Akumetsu-Dev** [comment]
Same problem: 
Installing “script@3.8.3” failed.Hide output…

npm ERR! git clone C:\Users\Walddys Emmanuel\.atom\.apm\_git-remotes\git-github-com-monsterkodi-node-simple-plist-git-6a71a541 C:\Users\WALDDY~1\AppData\Local\Temp\npm-3132-1591c5ac\git-cache-efb727829eab\5ff35d61ff600391ccd784a18616fa73c09e0aea: Cloning into 'C:\Users\WALDDY~1\AppData\Local\Temp\npm-3132-1591c5ac\git-cache-efb727829eab\5ff35d61ff600391ccd784a18616fa73c09e0aea'...
npm ERR! git clone C:\Users\Walddys Emmanuel\.atom\.apm\_git-remotes\git-github-com-monsterkodi-node-simple-plist-git-6a71a541 C:\Users\WALDDY~1\AppData\Local\Temp\npm-3132-1591c5ac\git-cache-efb727829eab\5ff35d61ff600391ccd784a18616fa73c09e0aea: error: cannot spawn sh: No such file or directory
npm ERR! git clone C:\Users\Walddys Emmanuel\.atom\.apm\_git-remotes\git-github-com-monsterkodi-node-simple-plist-git-6a71a541 C:\Users\WALDDY~1\AppData\Local\Temp\npm-3132-1591c5ac\git-cache-efb727829eab\5ff35d61ff600391ccd784a18616fa73c09e0aea: fatal: unable to fork
npm ERR! git clone C:\Users\Walddys Emmanuel\.atom\.apm\_git-remotes\git-github-com-monsterkodi-node-simple-plist-git-6a71a541 C:\Users\WALDDY~1\AppData\Local\Temp\npm-3132-1591c5ac\git-cache-efb727829eab\5ff35d61ff600391ccd784a18616fa73c09e0aea: 
npm ERR! Windows_NT 6.2.9200
npm ERR! argv "C:\\Users\\Walddys Emmanuel\\AppData\\Local\\atom\\app-1.9.2\\resources\\app\\apm\\bin\\node.exe" "C:\\Users\\Walddys Emmanuel\\AppData\\Local\\atom\\app-1.9.2\\resources\\app\\apm\\node_modules\\npm\\bin\\npm-cli.js" "--globalconfig" "C:\\Users\\Walddys Emmanuel\\.atom\\.apm\\.apmrc" "--userconfig" "C:\\Users\\Walddys Emmanuel\\.atom\\.apmrc" "install" "C:\\Users\\WALDDY~1\\AppData\\Local\\Temp\\d-11674-8092-v44pho\\package.tgz" "--target=0.37.8" "--arch=ia32"
npm ERR! node v0.10.40
npm ERR! npm  v2.13.3
npm ERR! code 128

npm ERR! Command failed: Cloning into 'C:\Users\WALDDY~1\AppData\Local\Temp\npm-3132-1591c5ac\git-cache-efb727829eab\5ff35d61ff600391ccd784a18616fa73c09e0aea'...
npm ERR! error: cannot spawn sh: No such file or directory
npm ERR! fatal: unable to fork
npm ERR! 
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

i reinstall all, include the OS

**Comment by genghuiluo** [comment]
@Akumetsu-Dev , [FYI]!(https://github.com/atom/apm/issues/322)
To be clear: this is fixed in apm versions >= 1.11.0. Here is the commit that fixes it. Note the tags and branches of apm the commit is contained in. You can run apm --version to see if you have a version of apm with this fix in it. Note that currently, Atom does not come bundled with a version of apm that has this fix. The latest beta release of Atom uses apm 1.10.0. Once atom/atom#11897 is merged and released, Atom will include a version of apm with this fix.

Note also that even after this is fixed, you're still going to need to tell apm about your proxy, as explicitly stated in the README.

**Comment by Akumetsu-Dev** [comment]
Hello, i found the solution, first GitHub windows Client not installed completely, what i mean the next step below, the installation process had to do it automatic, but did not.

1- C:\Users\<NameofUser>\AppData\Local\GitHub\PortableGit_<Serial>, in this folder you will find a batch named post-install you have to install it with admin permission.

2- Add to the environment variable the next path:
- C:\Users\<NameofUser>\AppData\Local\GitHub\PortableGit_PortableGit_<Serial>\usr\bin
- C:\Users\<NameofUser>\AppData\Local\GitHub\PortableGit_PortableGit_<Serial>\etc\ssh
- C:\Users\<NameofUser>\AppData\Local\GitHub\PortableGit_PortableGit_<Serial>\cmd
                    *GitHub didnt do this after the instalation, you have to do it manual.
![image](https://cloud.githubusercontent.com/assets/7697566/17557424/248b2978-5ee5-11e6-8e68-36bea6692561.png)

3- In the C:\Users\<NameofUser>\AppData\Local\GitHub\PortableGit_PortableGit_<Serial>\cmd, you will find 2 *.cmd files: "start-ssh-agent" and "start-ssh-pageant" run in with admin permission.
![image](https://cloud.githubusercontent.com/assets/7697566/17557449/3e3469ac-5ee5-11e6-94a6-2054c0fd363a.png)

4- Reboot windows and try install script.
